### PR TITLE
Add Veracity Docs to Developer Patterns

### PR DESCRIPTION
**Pull request by henry739**
This PR adds user-friendly (hopefully!) documentation for the `verify-inclusion` and `node` commands of the Veracity tool. Looking for feedback on the accuracy (@robinbryce) and also the messaging, clarity and tone (@JAG-UK )

Layout example:
image

re [AB#9653](https://dev.azure.com/jitsuin/0629f48c-3979-4bbc-9026-cb06b3dfd0ae/_workitems/edit/9653)

### CODE REVIEW

File: content/developers/developer-patterns/veracity/index.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+DataTrails records the events that matter to your business and lets you prove them at a later date. 
+This guide will show how to do this for both online and offline data scenarios. 
+
+For simplicity we'll walk through an example of proving that a publicly attested event exists on the 
+merkle log for the public tenant on DataTrails. If you want to try this with your own data, simply
+download a copy of your event from the DataTrails API and supply your tenant ID instead of the public
+one.
+
+#### Setup
+Lets set some variables that reference the public tenant in DataTrails and a public event that
```
Comment (eccles) [inline content/developers/developer-patterns/veracity/index.md:43]: Let's

File: content/developers/developer-patterns/veracity/index.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+    verify-included
+```
+
+```sh
+...
+verifying for tenant: tenant/6ea5cd00-c711-3649-6914-7b125928bbb4
+verifying: 5772 2889 01917aeb9103048500 publicassets/046ad7b4-dc99-4f90-9511-d2fad2e72bed/events/fef3c753-52e5-406b-8e41-8a36a2cc4818
+XX|5772 2889
+
+error: the entry is not in the log. for tenant tenant/6ea5cd00-c711-3649-6914-7b125928bbb4
```
Comment (eccles) [inline content/developers/developer-patterns/veracity/index.md:128]: garbled?

File: content/developers/developer-patterns/veracity/index.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+#### Offline Verification
+Veracity can be used to verify the inclusion of an event in an offline backup of a DataTrails 
+merkle log. We can do this by supplying a `--data-local` argument instead of `--data-url`. First, 
+we'll need to get a copy of the massif.
+
+{{< note >}}
+**Note:** DataTrails break the merkle log down into manageable chunks called massifs. Once each massif
+is full, a new one is started. The filenames are numbered (e.g. 0000000000000000.log, 0000000000000001.log) to indicate order. 
+
+The argument `--data-local` accepts a single massif file or a directory containing multiple. The event
```
Comment (eccles) [inline content/developers/developer-patterns/veracity/index.md:142]: multiple what?

Comment (henry739) [reply content/developers/developer-patterns/veracity/index.md:142]: multiple massif files

**Review by robinbryce — APPROVED** [review]
Lovely, thanks!

File: content/developers/developer-patterns/veracity/index.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
+---
```
Comment (henry739) [inline content/developers/developer-patterns/veracity/index.md:1]: Re-order so that this comes just before navigating merklelogs

**Review by robinbryce — APPROVED** [review]
LGTM

**Review by JAG-UK — APPROVED** [review]

**Pull request merged**